tests/epyccel/test_arrays.py

```diff
@@ -3516,322 +2774,160 @@ def test_array_2d_F_slice_stride_23(language):
     f2 = epyccel(f1, language = language)
     assert np.array_equal(f1(a), f2(a))
 
-@pytest.mark.parametrize( 'language', [
-        pytest.param("c", marks = [
-            pytest.mark.skip(reason="sum and len not implemented in c"),
-            pytest.mark.c]),
-        pytest.param("fortran", marks = pytest.mark.fortran)
-    ]
-)
 def test_array_2d_C_slice_stride_10(language):
     a = arrays.a_2d_c
 
     f1 = arrays.array_2d_C_slice_stride_10
     f2 = epyccel(f1, language = language)
     assert np.array_equal(f1(a), f2(a))
 
-@pytest.mark.parametrize( 'language', [
-        pytest.param("c", marks = [
-            pytest.mark.skip(reason="sum and len not implemented in c"),
-            pytest.mark.c]),
-        pytest.param("fortran", marks = pytest.mark.fortran)
-    ]
-)
 def test_array_2d_C_slice_stride_11(language):
     a = arrays.a_2d_c
 
     f1 = arrays.array_2d_C_slice_stride_11
     f2 = epyccel(f1, language = language)
     assert np.array_equal(f1(a), f2(a))
 
-@pytest.mark.parametrize( 'language', [
-        pytest.param("c", marks = [
-            pytest.mark.skip(reason="sum and len not implemented in c"),
-            pytest.mark.c]),
-        pytest.param("fortran", marks = pytest.mark.fortran)
-    ]
-)
 def test_array_2d_C_slice_stride_12(language):
     a = arrays.a_2d_c
 
     f1 = arrays.array_2d_C_slice_stride_12
     f2 = epyccel(f1, language = language)
     assert np.array_equal(f1(a), f2(a))
 
-@pytest.mark.parametrize( 'language', [
-        pytest.param("c", marks = [
-            pytest.mark.skip(reason="sum and len not implemented in c"),
-            pytest.mark.c]),
-        pytest.param("fortran", marks = pytest.mark.fortran)
-    ]
-)
 def test_array_2d_C_slice_stride_13(language):
     a = arrays.a_2d_c
 
     f1 = arrays.array_2d_C_slice_stride_13
     f2 = epyccel(f1, language = language)
     assert np.array_equal(f1(a), f2(a))
 
-@pytest.mark.parametrize( 'language', [
-        pytest.param("c", marks = [
-            pytest.mark.skip(reason="sum and len not implemented in c"),
-            pytest.mark.c]),
-        pytest.param("fortran", marks = pytest.mark.fortran)
-    ]
-)
 def test_array_2d_C_slice_stride_14(language):
     a = arrays.a_2d_c
 
     f1 = arrays.array_2d_C_slice_stride_14
     f2 = epyccel(f1, language = language)
     assert np.array_equal(f1(a), f2(a))
 
-@pytest.mark.parametrize( 'language', [
-        pytest.param("c", marks = [
-            pytest.mark.skip(reason="sum and len not implemented in c"),
-            pytest.mark.c]),
-        pytest.param("fortran", marks = pytest.mark.fortran)
-    ]
-)
 def test_array_2d_C_slice_stride_15(language):
     a = arrays.a_2d_c
 
     f1 = arrays.array_2d_C_slice_stride_15
     f2 = epyccel(f1, language = language)
     assert np.array_equal(f1(a), f2(a))
 
-@pytest.mark.parametrize( 'language', [
-        pytest.param("c", marks = [
-            pytest.mark.skip(reason="sum and len not implemented in c"),
```

Comment:
Is `len` implemented in C now? (If it isn't I think it's a 1or 2 line fix so probably worth adding here)

Reply:
I think it is impelemented! i tested it and it worked! I activated these tests too.

Reply:
Great! Apparently I did that ^^'. I obviously forgot to update the skip reasons.